File: 백준/Silver/14425. 문자열 집합/문자열 집합.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.ArrayList;
import java.util.List;

public class Main {
  public static void main(String[] args) throws IOException {
    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    String[] s = bufferedReader.readLine().split(" ");
    List<String> list = new ArrayList<>();
    int N = Integer.parseInt(s[0]);
    int M = Integer.parseInt(s[1]);
    int count = 0;
    for(int i =0; i< N; i++){
        list.add(bufferedReader.readLine());
    }
    for(int j = 0; j < M; j++){
      String s1 = bufferedReader.readLine();
      if(list.contains(s1)){
        count++;
      }
    }
    System.out.println(count);
  }
}

File: 백준/Silver/14425. 문자열 집합/문자열 집합.py
import sys

N, M = map(int, sys.stdin.readline().strip().split())
sumOfStrings = []
inputList = []
count = 0

for _ in range(N):
    sumOfStrings.append(sys.stdin.readline().strip())

for i in range(M):
    inputList.append(sys.stdin.readline().strip())

    if inputList[i] in sumOfStrings:
        count+=1

print(count)